middle_end/flambda2/types/flambda2_types.mli

```diff
@@ -606,6 +628,8 @@ type boxed_or_tagged_number = private
 val prove_is_a_boxed_or_tagged_number :
   Typing_env.t -> t -> boxed_or_tagged_number proof_of_property
 
+val prove_nothing : Typing_env.t -> t -> _ proof_of_property
```

Comment:
I don't think it makes sense to expose this function. It doesn't seem to be used at all.